src/integrationTest/java/uk/gov/hmcts/reform/jps/repository/CourtVenueRepositoryTest.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.hmcts.reform.jps.repository;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.jps.domain.CourtVenue;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@DataJpaTest
+@ActiveProfiles("itest")
+class CourtVenueRepositoryTest {
+
+    @Autowired
+    private CourtVenueRepository courtVenueRepository;
+
+    @BeforeEach
+    void setUp() {
+        courtVenueRepository.deleteAll();
+    }
+
+    @Test
+    void shouldSaveSittingRecord() {
+        CourtVenue courtVenue = createCourtVenue();
+
+        CourtVenue persistedCourtVenue = courtVenueRepository.save(courtVenue);
+
+        assertThat(persistedCourtVenue).isNotNull();
+        assertThat(persistedCourtVenue.getId()).isNotNull();
+        assertThat(persistedCourtVenue).isEqualTo(courtVenue);
+    }
+
+    private CourtVenue createCourtVenue() {
+        return CourtVenue.builder()
+            .epimmsId("1234")
+            .hmctsServiceId("test")
+            .costCenterCode("456")
+            .build();
+    }
+
+    @Test
+    void shouldUpdateSittingRecordWhenRecordIsPresent() {
```

Comment:
```suggestion
    void shouldUpdateCourtVenueWhenRecordIsPresent() {
```

Reply:
Updated to shouldUpdateCourtVenueWhenRecordIsPresent